api/controllers/AuthController.js

```diff
@@ -21,7 +21,6 @@ module.exports = {
   },
 
   reddit: function(req, res) {
-    console.log(req);
```

Comment:
Ah yes, i noticed there were so many logs. Any other console logs lying around? I should add that to travis to shout at us if we have it.